sda-download/api/s3/s3.go

```diff
@@ -294,7 +333,11 @@ func Download(c *gin.Context) {
 		ListBuckets(c)
 
 	case c.Param("filename") != "":
-		GetObject(c)
+		if strings.Contains(c.Request.URL.String(), "encrypted") {
```

Comment:
So, downloading a file called `this-is-not-encrypted` should always trigger encrypted objects? 

Probably fine with something like `strings.HasPrefix(c.Request.URL.Path, "/s3-encrypted")` but haven't tested.

Reply:
:see_no_evil:
Good catch. Fixed in 6db0315.